### PR TITLE
Add 1.1 to the roadmap.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

**Comment by brendandburns** [comment]
@bgrant0607 @thockin @smarterclayton

**Comment by bgrant0607** [comment]
I'm fine with this, but this doc is a PITA to keep up to date. For example, it's going to be copied into the 1.1 branch when we cut the branch, and then will get copied to gh_pages. Do we want it on kubernetes.io after 1.1 is released? It should be superseded by the release notes, IMO.

I updated the v1.1 milestone description with the P1s. We could add the rest of this there, as well. Or, we could put it on a wiki page and drop a link to it in the milestone description, since there's no way I can tell to link directly to the milestone description.

**Comment by thockin** [comment]
What if we put this in the gh-pages branch instead of master?

**Comment by brendandburns** [comment]
@thockin where do you think it should go in gh-pages?

I would like to have a concrete page to point to, the milestone list is good too, but it's too detailed.

File: docs/roadmap.md [hunk 1]
```diff
@@ -33,7 +33,46 @@ Documentation for other releases can be found at
...
+
+### Scope
+
+The 1.1 release of Kubernetes will be a purely additive releases, the `v1` API will be maintained, with a set
+of newly added features.
+
+#### Blocking Features
+
+The following features are considered blocking for the 1.1 release:
+   * Docker 1.8.x
```
Comment (dchen1107) [inline docs/roadmap.md]: Current plan is 1.8.2

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**